parastell/invessel_build.py

```diff
@@ -299,20 +299,24 @@ def import_step_cubit(self):
             vol_id = cubit_io.import_step_cubit(name, self.export_dir)
             data["vol_id"] = vol_id
 
-    def export_step(self, export_dir=""):
-        """Export CAD solids as STEP files via CadQuery.
+    def export_components(self, filetype="step", export_dir=""):
+        """Export CAD solids as STEP or STL files via CadQuery.
```

Comment:
It might be a good idea to include some kind of check on filetype input, unless we want to rely on CadQuery to issue an error if an incorrect filetype is supplied.